### ISSUE TITLE
Some times while creation of project, scenario or masterclass, the wrong price value automatically comes in Price input

### ISSUE THREAD

**Issue by Burhanuddin-Nahargarwala** [enqurious/enqurious-platforms-bug-reporting#40]
I haven't added any value in the price input, but then also the given value automatically came:
![image](https://github.com/enqurious/enqurious-platforms-bug-reporting/assets/105622461/04701f91-5c95-4df1-8da9-35e1447f5d83)

This happened to me 2 to 3 times. At first, I thought that might be it is my mistake, and mistakenly the value might get added, but today also this occurred, that's why I thought to add it to the issue.

**Comment by mskl-kashish** [comment]
Hey @Burhanuddin-Nahargarwala, this might happen if you click on the input and scroll with your mouse, then the value appears.
is this what happens with you ?

**Comment by Burhanuddin-Nahargarwala** [comment]
Yes @mskl-kashish, actually I changed the input to Price and then scrolled down to check for other options. I think at that time the Price value might changed.

**Comment by Burhanuddin-Nahargarwala** [comment]
Got it. I am closing the issue. Thanks
